From 65ed09f5c8f6e326304322ce8fde90d703e64421 Mon Sep 17 00:00:00 2001
From: Alex Evraev <alexev@marvell.com>
Date: Sun, 28 Jun 2020 11:21:25 +0300
Subject: [PATCH 1187/1239] net: mvpp2: fix comphy4 phy selector bits offset

Fixed "comphy4 phy selector" bits offset in
Common PHYs PHY Selectors register

Signed-off-by: Alex Evraev <alexev@marvell.com>
Change-Id: Id3b35af3df271965fb8ff542cae1ea1c97378c2d
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/boot/u-boot/+/31006
Reviewed-by: Stefan Chulski <stefanc@marvell.com>
Reviewed-by: Kostya Porotchkin <kostap@marvell.com>
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
---
 drivers/net/mvpp2.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/mvpp2.c b/drivers/net/mvpp2.c
index 7c00c88787..f1180ae2e2 100644
--- a/drivers/net/mvpp2.c
+++ b/drivers/net/mvpp2.c
@@ -76,7 +76,7 @@ do {									\
 #define COMPHY2_SEL_BIT_OFFS			8
 #define COMPHY4_SEL_BIT_OFFS			16
 #define COMPHY2_SEL_MASK			0xF00
-#define COMPHY4_SEL_MASK			0xF000
+#define COMPHY4_SEL_MASK			0xF0000
 #define COMPHY2_MODE_CHECK(val)						      \
 			(((val) & COMPHY2_SEL_MASK) >> COMPHY2_SEL_BIT_OFFS)
 #define COMPHY4_MODE_CHECK(val)						      \
-- 
2.29.0

